packages/gui/src/components/AddProperty.tsx

```diff
@@ -24,10 +25,9 @@ export const AddPropertyControl = ({
   const id = useId()
   const inputRef = useRef(null)
 
-  // @ts-ignore
-  const allProperties: string[] = Object.entries(propertyList)
+  const allProperties: (string | null)[] = Object.entries(propertyList)
     .map(([name, data]) => {
-      return data.input !== 'none' ? name : null
+      return data.input !== 'none' ? kebabCase(name) : null
```

Comment:
We want to kebab just in the list item, right? We still want the data to be camel cased when added to the style object. Only want the display to be kebab.

Reply:
Rather than re-camel casing we can `{kebabCase(item)}` around line 200.